options/locale/locale_en-US.ini

```diff
@@ -581,6 +581,8 @@ commits.message = Message
 commits.date = Date
 commits.older = Older
 commits.newer = Newer
+commits.signed_by = Signed by
+commits.gpg_key_id = GPG key ID
```

Comment:
@bkcsoft fixed, I was testing if it works correctly and doing undo/redo and forgot to save latest code changes in editor :)